hyfetch/__init__.py

```diff
@@ -1,9 +1,7 @@
-from __future__ import annotations
-
```

Comment:
Also please don't remove `from __future__ import annotations`, they're required for Python 3.7-8 compatibility. Thanks!